README.md

```diff
@@ -4,3 +4,7 @@ Comprehensive Haskell bindings to
 [GLPK](https://www.gnu.org/software/glpk/). No attempt is made to make
 these bindings conform to Haskell norms --- they are a direct
 translation from the GLPK header file.
+
+## Requirements
+
+* GLPK >=4.62
```

Comment:
Did you test this with 4.62? I have 4.65 installed, but I'd believe that 4.62 works as well.

Reply:
Thanks for pointing this out, and I hope you find the bindings useful!